### ISSUE TITLE
CSS Color indicator doesn't have a border on the high contrast theme

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#6463]
It makes it hard to see colors close to black.

![image](https://cloud.githubusercontent.com/assets/2193314/15346400/a6887c02-1c6c-11e6-8380-b6882a34f6ad.png)

**Comment by bpasero** [comment]
Seems fixed:

![image](https://cloud.githubusercontent.com/assets/900690/26462326/933dd662-4180-11e7-8f1a-24ceaada2c14.png)
